doc/source/management/pam_plugin.rst

```diff
@@ -72,6 +72,11 @@ Supplementary groups support
 
 This feature enables using secondary groups in the mapping part of the authentication string, like "``mysql, developers=joe, dba=mark``". Previously only primary groups could have been specified there. If user is a member of both ``developers`` and ``dba``, PAM plugin will map it to the ``joe`` because ``developers`` matches first.
 
+Known issues
+============
+
+Default mysql stack size in not enough to handle ``pam_ecryptfs`` module. Workaround is to increase the |MySQL| stack size to ``262144`` by setting the `thread-stack <https://dev.mysql.com/doc/refman/5.5/en/server-system-variables.html#sysvar_thread_stack>`_ variable.
```

Comment:
Hmm not perfect neither. The default is is 256KB too, so if thread_stack_size is not set, the advice would be to set it to 512KB, otherwise increase its old value by 256KB. Not sure how best to phrase this.